File: combate-mortal/src/main/java/com/combatem/Escudo.java
package com.combatem;

public class Escudo {
    //final para asignar
    private final double porcentajeProteccion; 

    public Escudo(double porcentajeProteccion) {
        this.porcentajeProteccion = porcentajeProteccion;
    }

    public double modificarDanio(double danio) {
        return danio * (1 - porcentajeProteccion);
    }
}

File: py-combate-mortal/engine/escudo.py
from .unidad import Unidad

class Escudo:
    def __init__(self, porcentaje_proteccion):
        self.porcentaje_proteccion = porcentaje_proteccion

    def modificar_danio(self, danio):
        return danio * (1 - self.porcentaje_proteccion)